source/brailleDisplayDrivers/papenmeier_serial.py

```diff
@@ -182,11 +182,11 @@ def _handleKeyPresses(self): #called by the keycheck timer
 		"""if a button was pressed an input gesture is executed"""
 		if(self._dev!=None):
 			data = brl_poll(self._dev)
-			if(len(data) == 10 and data[1]=='K'):
-				pos = ord(data[2])*256+ord(data[3])
+			if len(data) == 10 and data[1] == ord('K'):
+				pos = data[2] * 256 + data[3]

```

Comment:
```suggestion
				pos = data[2] << 8 | data[3]
```

Reply:
Probably better to use `int.from_bytes` to convert this, the intention is clearer. That said, I'm going to leave it as is. It's not required to change to Python 3.